lib/auth0/api/authentication_endpoints.rb

```diff
@@ -90,10 +90,11 @@ def change_password(email, password, connection_name = UP_AUTH)
       def start_passwordless_email_flow(email, send = 'link', auth_params = {})
         raise Auth0::InvalidParameter, 'Must supply a valid email' if email.to_s.empty?
         request_params = {
-          client_id:    @client_id,
-          email:        email,
-          send:         send,
-          auth_params:  auth_params
+          client_id:   @client_id,
+          connection:  'email',
+          email:       email,
+          send:        send,
+          authParams:  auth_params
```

Comment:
In the documentation the parameter is authParams (https://auth0.com/docs/api/authentication#!#post--with_email)